src/main/kotlin/study/racing/domain/Cars.kt

```diff
@@ -0,0 +1,54 @@
+package study.racing.domain
+
+import study.racing.strategy.MoveStrategy
+import study.racing.strategy.RandomMoveStrategy
+
+class Cars private constructor(
+    private var cars: List<Car> = listOf(),
```

Comment:
```suggestion
    private val cars: List<Car>,
```

var -> val로 변경해보는 건 어떨까요?